lms/static/js/spec/dashboard/track_events_spec.js

```diff
@@ -14,70 +14,51 @@
+                                           window.edx.dashboard.generateTrackProperties];
 
-            it('sends an analytics event when the user clicks enter course link', function() {
-                // Verify that analytics events fire when the options dropdown is engaged.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.wrapper-action-more'),
-                    'edx.bi.dashboard.course_options_dropdown.clicked',
-                    window.edx.dashboard.generateTrackProperties
-                );
-            });
+                var courseOptionDropdownClickedArgs = [$('.wrapper-action-more'),
+                                                       'edx.bi.dashboard.course_options_dropdown.clicked',
+                                                       window.edx.dashboard.generateTrackProperties];
 
-            it('sends an analytics event when the user clicks the learned about verified track link', function() {
-                //Verify that analytics events fire when the 'Learned about verified track' link is clicked.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.verified-info'),
-                    'edx.bi.dashboard.verified_info_link.clicked',
-                    window.edx.dashboard.generateTrackProperties
-                );
-            });
+                var verifiedInfoLinklArgs = [$('.verified-info'),
+                                             'edx.bi.dashboard.verified_info_link.clicked',
+                                             window.edx.dashboard.generateTrackProperties];
 
-            it('sends an analytics event when the user clicks find courses button', function() {
-                // Verify that analytics events fire when the 'user clicks find the course' button.
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.btn-find-courses'),
-                    'edx.bi.dashboard.find_courses_button.clicked',
-                    {
-                        category: 'dashboard',
-                        label: null
-                    }
-                );
-            });
+                var findCourseButtonArgs = [$('.btn-find-courses'),
+                                            'edx.bi.dashboard.find_courses_button.clicked',
+                                            {
+                                                category: 'dashboard',
+                                                label: null
+                                            }];
 
-            it('sends an analytics event when the user clicks the \'View XSeries Details\' button', function() {
-                expect(window.analytics.trackLink).toHaveBeenCalledWith(
-                    $('.xseries-action .btn'),
-                    'edx.bi.dashboard.xseries_cta_message.clicked',
-                    window.edx.dashboard.generateProgramProperties
-                );
```

Comment:
If I had to guess, I'd guess that the `HTMLNode` objects weren't evaluating equal to the spy, perhaps because of a timing problem.

This is the expected call

    [
        {
            0 : HTMLNode,
            length : 1,
            prevObject : { 0 : HTMLNode, context : HTMLNode, length : 1 },
            context : HTMLNode,
            selector : '.course-title > a'
        },
        'edx.bi.dashboard.course_title.clicked',
        Function
    ]

one of the elements in the list of actual calls is

    [
        {
            0 : HTMLNode,
            length : 1,
            prevObject: { 0 : HTMLNode, context : HTMLNode, length : 1 },
            context : HTMLNode,
            selector : '.course-title > a'
        },
        'edx.bi.dashboard.course_title.clicked',
        Function
    ]

The only places that could be different are the various HTMLNodes and the Function calls (neither of which have any identifying info in the print statement).